src/Microsoft.ML.PipelineInference/Microsoft.ML.PipelineInference.csproj

```diff
@@ -7,6 +7,10 @@
     <DefineConstants>CORECLR</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>;1591;0618</NoWarn>
+  </PropertyGroup>
```

Comment:
Could we clarify this? What warnings are these, and what is the relationship to your change?

Reply:
I had added 0618 but now I have removed it and applied to code sections.

---
In reply to: [188006377](https://github.com/dotnet/machinelearning/pull/142#discussion_r188006377) [](ancestors = 188006377)